addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -210,6 +187,93 @@ odoo.define('website_slides.quiz', function (require) {
             });
         },
 
+        _animateProgressBar: function (self, $modal, rankProgress) {
```

Comment:
You don't need to pass 'self' to the submethod.

In the submethod, the scope of "this" will be equal to "self" of the caller.

If that's not clear, I can explain "de vive voix".